lib/web_ui/dev/common.dart

```diff
@@ -54,6 +58,22 @@ class _LinuxBinding implements PlatformBinding {
   @override
   String getChromeExecutablePath(io.Directory versionDir) =>
       path.join(versionDir.path, 'chrome-linux', 'chrome');
+
+  @override
+  String getFirefoxDownloadUrl(String version) {
+    return 'https://download-installer.cdn.mozilla.net/pub/firefox/releases/${version}/linux-x86_64/en-US/firefox-${version}.tar.bz2';
+  }
+
+  @override
+  String getFirefoxExecutablePath(io.Directory versionDir) {
+    // TODO: implement getFirefoxExecutablePath
+    return null;
```

Comment:
Maybe throw `UnimplementedError`?

Reply:
will be implemented soon. This is not a customer facing method.